canvas_modules/common-canvas/src/common-properties/controls/passwordfield/passwordfield.jsx

```diff
@@ -34,6 +35,14 @@ class PasswordControl extends React.Component {
 	}
 
 	render() {
+		const overrideLabelKeyShow = `${this.props.control.name}.passwordShow.tooltip`;
+		const overrideLabelKeyHide = `${this.props.control.name}.passwordHide.tooltip`;
```

Comment:
`overrideLabelKeyShow` can be renamed to `overrideShowPasswordLabel`
`overrideLabelKeyHide` can be renamed to `overrideHidePasswordLabel`